src/quo2/components/loaders/skeleton_list/view.cljs

```diff
@@ -0,0 +1,106 @@
+(ns quo2.components.loaders.skeleton-list.view
+  (:require [quo2.theme :as theme]
+            [react-native.core :as rn]
+            [quo2.foundations.colors :as colors]
+            [react-native.masked-view :as masked-view]
+            [react-native.reanimated :as reanimated]
+            [quo2.components.loaders.skeleton-list.style :as style]
+            [reagent.core :as reagent]
+            [quo2.components.loaders.skeleton-list.constants :as constants]))
+
+(defn skeleton-label
+  [animated? index parent-index]
+  (str (if animated? "skeleton-animated-" "skeleton-static-")
+       parent-index
+       "-"
+       index))
+
+(defn static-skeleton-view
+  [{:keys [index parent-index content color]}]
+  [rn/view
+   {:style               (style/container content)
+    :accessibility-label (skeleton-label false index parent-index)}
+   [rn/view {:style (style/avatar color)}]
+   [rn/view {:style style/content-container}
+    [rn/view
+     {:style (style/content-view
+              {:type    (if (= content :list-items) :message :author)
+               :index   index
+               :content content
+               :color   color})}]
+    [rn/view
```

Comment:
Hi, we are missing when condition here, the second content view is only present for `:notifications` type skeleton. Also this view is currently not visible for any content type
<img src="https://github.com/status-im/status-mobile/assets/17097240/3045a392-32a5-41a7-b956-e23913f1d9d2" height=200/>